docker-compose-lts.yml

```diff
@@ -0,0 +1,75 @@
+version: "3.8"
+services:
+  iroha0:
+    image: hyperledger/iroha2:lts
```

Comment:
if this is the only difference, could we not load it from an env variable?

Reply:
it depends whether you think docker files would change in a backward incompatible way. If so then duplication is favored IMO

Reply:
I think that we can afford to go with such an approach for now, and a generation tool shouldn't be that hard to implement if that's the only option.